Source/LibationFileManager/InteropFactory.cs

```diff
@@ -115,12 +117,28 @@ private static List<ProcessModule> LoadModuleList(string exeName)
         private static Assembly CurrentDomain_AssemblyResolve(object sender, ResolveEventArgs args)
         {
             // e.g. "System.Windows.Forms, Version=6.0.2.0, Culture=neutral, PublicKeyToken=b77a5c561934e089"
-            var asmName = args.Name.Split(',')[0];
+            var asmName = args.Name.Split(',')[0] + ".dll";
 
-            // `First` instead of `FirstOrDefault`. If it's not present we're going to fail anyway. May as well be here
-            var module = ModuleList.First(m => m.ModuleName.StartsWith(asmName));
+#if DEBUG
+
+            var modulePath = ModuleList.SingleOrDefault(m => m.ModuleName.EqualsInsensitive(asmName))?.FileName;
```

Comment:
If the `#else` code works in `DEBUG` also, it seems unnecessary to have special code for the `DEBUG` pragma

Reply:
It does if all dependencies copied to the same debug directory. That's a debug environment thing.  We can trim it if you like.

Reply:
If they aren't all copied the same dir then I don't think the debug can find it anyway. Or am I wrong?

Reply:
I just went ahead and commented out the hot loading code.

Reply:
Last question: If they aren't all copied the same dir then I don't think the debug can find it anyway. Or am I wrong?

Reply:
If you have the dotnet runtime installed on your machine (which of course you do if you're running VS), then the resolver will ask the runtime for the assembly name (which is in ProgramFiles). Otherwise, if it's not in the current directory or in %path%, it won't be found.

Reply:
As an example, create a new .net6 winforms app and run it. Look inside the debug directory and you won't find any dependencies. By default, VS does not copy all dependencies to the debug directory.

Reply:
Oh, I get it. This is about the GAC et al. -- my mind was stuck thinking about just the AppConfig. Hmm, good point. My debug builds won't be the same as the all-inclusive publishes.

I think I like it this best in its current form: commented out with good comments